src/main/java/com/osiris/autoplug/client/Main.java

```diff
@@ -270,6 +283,41 @@ public static void main(String[] _args) {
             CON = new ConMain();
             CON.open();
 
+
+            sshThread = new Thread(() -> {
+                try {
+                    sshServerSetup.start();
+                } catch (Exception e) {
+                    AL.warn("Failed to start SSH server", e);
+                }
+            });
+
+            boolean ssh_enabled = sshConfig.enabled.asBoolean();
+            if (ssh_enabled) {
+                AL.info("Starting SSH thread");
+                sshThread.start();
+            }
+
+            // Console output capture thread
+            capturer = new ConsoleOutputCapturer();
+            capturer.start();
+            new Thread(() -> {
+                while (true) {
+                    try {
+                        String newOutput = capturer.getNewOutput();
+                        if (!newOutput.isEmpty()) {
+                            // Other stuff that requires a clone of the console can go here
+                            if (ssh_enabled) {
+                                SSHServerConsoleReceive.broadcastToAll(newOutput);
+                            }
+                        }
+                        Thread.sleep(500); // Update the output every _ ms | 500 to minimize CPU usage
+                    } catch (InterruptedException e) {
+                        Thread.currentThread().interrupt();
+                    }
+                }
+            }).start();
```

Comment:
should also only be started if enabled

Reply:
this was intentional, so other services could utilize if they needed a copy of the console. Will implement tho. remove the check if other services want to use